docs/rules.md

```diff
@@ -174,7 +175,30 @@ let rule = new Rule({
 })
 ```
 
-Notice in the second example how `all`, `any`, and 'not' can be nested within one another to produce complex boolean expressions.  See the [nested-boolean-logic](../examples/02-nested-boolean-logic.js) example.
+Notice in the second example how `all`, `any`, and `not` can be nested within one another to produce complex boolean expressions.  See the [nested-boolean-logic](../examples/02-nested-boolean-logic.js) example.
+
+### Shared Conditions
+
+Rules my reference shared conditions based on their name.
+
+```js
+let rule = new Rule({
+  conditions: {
+    all: [
+      { uses: 'sharedCondition' },
+      { /* additional condition */ }
+    ]
+  }
+})
+```
+
+Before running the rule the shared condition should be added to the engine.
+
+```js
+engine.addSharedCondition('sharedCondition', { /* conditions */ });
+```
+
+Shared conditions must start with `all`, `any`, `not`, or reference a shared condition.
```

Comment:
nit: add a test or example of a shared condition referencing another shared condition